codalab/worker/worker_run_state.py

```diff
@@ -277,8 +320,13 @@ def mount_dependency(dependency, shared_file_system):
                         "Please ensure the shared fileystem between the server and "
                         "your worker is mounted properly or contact your administrators."
                     )
-                    logger.error(message)
+                    logger.error(
```

Comment:
Use the util function here

Reply:
parameter reason would be the first line.